lib/ssl/test/ssl_handshake_SUITE.erl

```diff
@@ -128,7 +128,7 @@ decode_hello_handshake(_Config) ->
 		    16#00, 16#00, 16#33, 16#74, 16#00, 16#07, 16#06, 16#73,
 		    16#70, 16#64, 16#79, 16#2f, 16#32>>,
 	
-    Version = {3, 0},
+    Version = ?'SSL-3.0',
```

Comment:
This is also testing SSL-3.0, I think we should aim to remove this suite all together. Thought so for a while, just have not have time to check that all that it test is actually covered by the property tests. It should be, but we  should check!